packages/deprecation-crawler/sandbox/deprecations/raw-deprecations.json

```diff
@@ -14,12 +14,13 @@
                 311
             ],
             "version": "master",
-            "remoteUrl": "https://github.com/timdeschryver/find-deprecations.git\n",
```

Comment:
This change is here because the git command in sandbox mode return an empty string
I also verified the line break at the end of the remoteUrl is removed.

A real run gives us:

```
{
            "path": "src/internal/ReplaySubject.ts",
            "lineNumber": 86,
            "name": "ReplaySubject._subscribe",
            "kind": "MethodDeclaration",
            "code": "...",
            "pos": [
                3598,
                3685
            ],
            "version": "master",
            "remoteUrl": "https://github.com/ReactiveX/rxjs",
            "date": "2020-06-16 09:44:20 -0500",
            "ruid": "3064759287"
        },
```